validator/sawtooth_validator/consensus/proxy.py

```diff
@@ -26,11 +29,15 @@ class ConsensusProxy:
     """Receives requests from the consensus engine handlers and delegates them
     to the appropriate components."""
 
-    def __init__(self, block_cache, chain_controller, block_publisher,
+    def __init__(self, block_cache, block_publisher,
+                 chain_controller, gossip, identity_signer,
                  settings_view_factory, state_view_factory):
         self._block_cache = block_cache
         self._chain_controller = chain_controller
         self._block_publisher = block_publisher
+        self._gossip = gossip
+        self._identity_signer = identity_signer
+        self._public_key = self._identity_signer.get_public_key  # hex, bytes?
```

Comment:
`self._identity_signer.get_public_key().as_bytes()` will give you bytes, `self._identity_signer.get_public_key().as_hex()` will give you hex

Reply:
You probably want bytes since that is what the consensus protos use.